dandi/cli/cmd_validate.py

```diff
@@ -36,19 +36,40 @@ def validate_bids(
         dandi validate-bids /my/path
     """
 
-    from ..bids_utils import is_valid, report_errors
+    from ..validate import Severity
     from ..validate import validate_bids as validate_bids_
 
-    validator_result = validate_bids_(
+    validator_result = validate_bids_(  # Controller
         *paths,
         report=report,
         report_path=report_path,
         schema_version=schema,
     )
-    valid = is_valid(validator_result)
-    report_errors(validator_result)
 
-    if not valid:
+    for i in validator_result:
```

Comment:
note: such "view" seems to be getting closer to our `validate` (each message separately) instead of what was in https://github.com/dandi/dandi-cli/pull/1104/files#diff-79d7645e39caaf3d1ca18c9ed87f99a1696696fa67da3464983fc07d6c974325L41 or how bids-validator presents groupped by type of error.  I guess it is ok, and we will provide alternative renderings later. But best to right away move this into some function e.g. `echo_individual_errors` or alike.

Reply:
should I extend and reuse e.g. `display_errors()` (in the same file)? It uses the logger instead of click.

Reply:
yeah - RF to reuse `display_errors()` for consistency with `validate`.

Next step after this PR would be to merry `validate` and `validate_bids` - we should have a single `validate` and there we will expose some option to switch between different "views" over errors.